File: 2475.数组中不等三元组的数目.py
#
# @lc app=leetcode.cn id=2475 lang=python3
#
# [2475] 数组中不等三元组的数目
#

# @lc code=start
class Solution:
    def unequalTriplets(self, nums: List[int]) -> int:
        res = 0
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                for k in range(j + 1, len(nums)):
                    if nums[i] != nums[j] and nums[j] != nums[k] and nums[i] != nums[k]:
                        res += 1
        return res
# @lc code=end

File: 2475.数组中不等三元组的数目.go
/*
 * @lc app=leetcode.cn id=2475 lang=golang
 *
 * [2475] 数组中不等三元组的数目
 */

// @lc code=start
func unequalTriplets(nums []int) int {
	res := 0
	for i := 0; i < len(nums); i++ {
		for j := i + 1; j < len(nums); j++ {
			for k := j + 1; k < len(nums); k++ {
				if nums[i] != nums[j] && nums[j] != nums[k] && nums[i] != nums[k] {
					res++
				}
			}
		}
	}
	return res

}

// @lc code=end

